docs/install_Device_Northbound.md

```diff
@@ -44,96 +44,94 @@ Instead of manually configuring you can also import the configuration files:
-certificates to allow a connection from the IIH.
+You need to have a Insights Hub account and create certificates to allow a connection from the Common Configurator.
 
 1. Configure the Databus credentials under "Settings > Databus credentials"
 
-  ![IIH_DatabusCredentials](graphics/IIH_DatabusCredentials.png)
+   ![IIH_DatabusCredentials](graphics/IIH3.png)
 
 2. Save the configuration
 
-3. Check if the device has an online connection in the "Home" tab, to proceed with configuring the Insights Hub connection.
-
-  ![IIH_OnlineStatus](graphics/IIH_OnlineStatus.png)
+3. Proceed with configuring the Insights Hub connection by clicking "Add" button.
 
-4. Click on "Add parent IED / Connect to MindSphere"
+   ![IIH_OnlineStatus](graphics/IIH4.png)
 
-  ![IIH_MindsphereConfig1](graphics/IIH_MindsphereConfig1.png)
-
-5. Select the device type "MindSphere Device"
-
-6. Enter credentials for the application. Those are required, to allow the IIH to interact with Insights Hubs REST-API.
-   The Insights Hub Tenant Administrator has to create them and assign the role "mdsp:core:Admin3rdPartyTechUser", to
-   allow the IIH to update the asset model in Insights Hub. More information can be found in the [IIH documentation](https://cache.industry.siemens.com/dl/dl-media/582/109803582/att_1087779/v6/EdgeApp_CommonConfigurator_en-US/en-US/index.html#treeId=487dc5ea471ae3fbdfb56d715301dad7) and [here](https://documentation.mindsphere.io/MindSphere/apps/operator-cockpit/application-credentials-for-API-applications.html) 
+4. Select the Target type. Enter credentials for the application. Those are required, to allow the IIH to interact with Insights Hubs REST-API.
+   
+   ![IIH_InsightsHub](graphics/IIH5.png)
 
-  ![IIH_MindsphereConfig2](graphics/IIH_MindsphereConfig2.png)
+   The Insights Hub Tenant Administrator has to create them and assign the role "mdsp:core:Admin3rdPartyTechUser", to allow the IIH to update the asset model in Insights Hub. More information can be found in the [IIH documentation](https://cache.industry.siemens.com/dl/dl-media/582/109803582/att_1087779/v6/EdgeApp_CommonConfigurator_en-US/en-US/index.html#treeId=487dc5ea471ae3fbdfb56d715301dad7) and [here](https://documentation.mindsphere.io/MindSphere/apps/operator-cockpit/application-credentials-for-API-applications.html)
 
-7. Enter the certificate details and upload the certificate and key. Information on how to create connector certificates
-   can be found [here](https://documentation.mindsphere.io/MindSphere/howto/howto-onboard-mindconnect-mqtt.html) and [here](https://documentation.mindsphere.io/MindSphere/howto/howto-managing-ca-certificates.html)
+   Enter the certificate details and upload the certificate and key. Information on how to create connector certificates can be found [here](https://documentation.mindsphere.io/MindSphere/howto/howto-onboard-mindconnect-mqtt.html) and [here](https://documentation.mindsphere.io/MindSphere/howto/howto-managing-ca-certificates.html)
```

Comment:
For user with least know how the prefered option to create certificates for MQTT can be found here:
https://documentation.mindsphere.io/MindSphere/howto/howto-obtaining-auto-generated-agent-certificate.html

Reply:
I added it, done